app/main/forms.py

```diff
@@ -1958,7 +1958,7 @@ class AdminEditEmailBrandingForm(StripWhitespaceForm):
         "Alt text", param_extensions={"hint": {"text": "Text for people who cannot see the logo"}}
     )
     colour = HexColourCodeField("Colour")
-    file = VirusScannedFileField("Upload a PNG logo", validators=[FileAllowed(["png"], "PNG Images only!")])
+    file = VirusScannedFileField("Upload a PNG logo", validators=[FileAllowed(["png"], "Logo must be a PNG file")])
```

Comment:
```suggestion
    file = VirusScannedFileField("Upload a PNG logo", validators=[FileAllowed(["png"], "The logo must be a PNG file")])
```

To match the content in row 100 of [the validation audit spreadsheet](https://docs.google.com/spreadsheets/d/1yeBQnz_M48W4QebzHp2Uq-M_D0u9hnXfjQGuRpJwYf8/edit?usp=sharing)